apps/learning-snippets/src/test/property-grid/ContextMenuItem.test.tsx

```diff
@@ -0,0 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/* eslint-disable no-console */
+/* eslint-disable import/no-duplicates */
+import { expect } from "chai";
+import { join } from "path";
+import sinon from "sinon";
+import { UiFramework, UiItemsManager } from "@itwin/appui-react";
+import { IModelReadRpcInterface, SnapshotIModelRpcInterface } from "@itwin/core-common";
+import { ECSchemaRpcInterface } from "@itwin/ecschema-rpcinterface-common";
+import { ECSchemaRpcImpl } from "@itwin/ecschema-rpcinterface-impl";
+import { PresentationRpcInterface } from "@itwin/presentation-common";
+import { Presentation } from "@itwin/presentation-frontend";
+import { HierarchyCacheMode, initialize as initializePresentationTesting, terminate as terminatePresentationTesting } from "@itwin/presentation-testing";
+// __PUBLISH_EXTRACT_START__ PropertyGrid.ExampleContextMenuItemImports
+import { PropertyGridContextMenuItem } from "@itwin/property-grid-react";
+import type { ContextMenuItemProps } from "@itwin/property-grid-react";
+// __PUBLISH_EXTRACT_START__ PropertyGrid.ExampleContextMenuItemRegisterImports
+import { PropertyGridComponent } from "@itwin/property-grid-react";
+// __PUBLISH_EXTRACT_END__
+import { queryByText, render, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { buildIModel, insertPhysicalElement, insertPhysicalModelWithPartition, insertSpatialCategory } from "../../utils/IModelUtils";
+import { PropertyGridTestUtils } from "../../utils/PropertyGridTestUtils";
+
+describe("Property grid", () => {
+  describe("Learning snippets", () => {
+    describe("Context menu item", () => {
+      const user = userEvent.setup();
```

Comment:
I think this should be moved to `it` block

Reply:
Moved